app/controllers/casework/crime_applications_controller.rb

```diff
@@ -1,10 +1,13 @@
 module Casework
   class CrimeApplicationsController < Casework::BaseController
+    include ApplicationSearchable
+
     before_action :set_crime_application, only: %i[show history complete ready]
```

Comment:
Small thing but a possible refactor might be to encapsulate these not restful names in their own controller like `HistoryController` and then just use a restful `show` method. 

Not a priority at the moment though.

Reply:
Good idea, let's take a look when we add the work stream tabs.